base_layer/core/src/chain_storage/lmdb_db/lmdb_db.rs

```diff
@@ -599,7 +617,9 @@ where D: Digest + Send + Sync
                 }
                 pruned_mmr
             },
-        })
+        });
+        trace!(target: LOG_TARGET, "pruned mmr requested: {:?}", result);
```

Comment:
With no context, this message adds no value. Besides, since you only log result, the caller could also log this.